packages/mui-utils/src/useOnMount/useOnMount.ts

```diff
@@ -7,6 +7,7 @@ const EMPTY = [] as unknown[];
  * A React.useEffect equivalent that runs once, when the component is mounted.
  */
 export default function useOnMount(fn: React.EffectCallback) {
+  // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler -- 'fn'. Either include it or remove the dependency array
```

Comment:
The explanation is not clear. Can you update it to make it clearer? Maybe something like:

```suggestion
  // TODO: uncomment once we enable eslint-plugin-react-compiler // eslint-disable-next-line react-compiler/react-compiler -- no need to put `fn` in the dependency array
```